en/services/component_information.md

```diff
@@ -130,14 +130,59 @@ In summary:
    - If not recieved the GCS should resend the request (typically in the application level).
    - Once information is received:
      - the GCS checks if `COMPONENT_INFORMATION.general_metadata_file_crc` matches its cached CRC value.
-	   If so, there is no need to download the [general metadata file](#COMP_METADATA_TYPE_GENERAL) (or other files it references) as it has not changed since the last download.
+       If so, there is no need to download the [general metadata file](#COMP_METADATA_TYPE_GENERAL) (or other files it references) as it has not changed since the last download.
 	 - the GCS checks if `COMPONENT_INFORMATION.peripherals_metadata_uri` if supplied, and (if so) whether the `peripherals_metadata_file_crc` field matches the cached value.
 	 
 	 If the cached values do not match the associated files should be downloaded and parsed ....
 1. GCS downloads the file specified in the `general_metadata_uri` using MAVLink FTP.
 1. GCS parses the general metadata for other supported metadata locations, and then downloads the files via MAVFTP or HTTP(s).
    This may be done immediately, or as needed.
 
+## Metadata Types
+
+### Actuators (COMP_METADATA_TYPE_ACTUATORS)
+
+The actuators metadata allows a GCS to create a UI to configure and test actuators, and configure vehicle geometries, without having to understand anything about the underlying flight stack.
+
+
+> **Note** The mechanism works similarly to [camera definition files](../services/camera_def.md).
+> It can be used for flight stacks that allow outputs and geometry definition to be configured using parameters.
+> If flight stack outputs or geometries cannot be configured using parameters, it can still be used to display the current geometry and outpu mappings and to test outputs.
```

Comment:
Is this true (I assume it is). If not, we should be precise about what this will and will not work with. For example, if you took out the new mixer stuff in PX4, what could you do with this metadata?

Reply:
It's correct